src/test/packages/51-import-everything/zarf.yaml

```
@@ -6,12 +6,14 @@ metadata:
 
 components:
   - name: import-component-local
+    description: "###ZARF_COMPONENT_NAME### case was here"
```

```suggestion
  - name: import-component-local
    description: "import-component-local == ###ZARF_COMPONENT_NAME###"
```

implements changes proposed above

---

this would be replicated across all of the components + the targets array

---

no problem making this change, just want to get your opinion on if you think this is needed as each component would all have the same ###ZARF_COMPONENT_NAME### as it is on a per component basis. The nature of the function using the string map of strings ensures this, or is this something that you want to unit test?